File: experiment/v1/order_of_if/negative_first.rb
def greeting(user)
  if !user[:title]
    name = user[:first_name]
  else
    name = "#{user[:title]} #{user[:last_name]}"
  end

  'Dear ' + name
end

user = {
  title: 'Miss',
  first_name: 'Jane',
  last_name: 'Marple'
}

result = greeting user

File: experiment/v1/order_of_if/negative_first.js
function getGreeting(user){
  var name;

  if(!user.title){
    name = user.firstName;
  }else{
    name = user.title + " " + user.lastName;
  }
  
  return "Dear " + name;
}

var user = {
  title: "Miss",
  firstName: "Jane",
  lastName: "Marple"
};

var result = getGreeting(user);